src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -1,66 +1,82 @@
-import React, { PureComponent, Fragment } from "react";
+import React, {PureComponent, Fragment} from "react";
 // Material UI Components
-import { withStyles, Tabs, Tab, Typography, Paper } from "@material-ui/core";
+import {withStyles, Tabs, Tab, Typography} from "@material-ui/core";
 // Components
 import MonitoringData from "./MonitoringData";
 import ChartContainer from "./ChartContainer";
-// import HistoryChart from "./historyChart";
 //Style
-import { dashboardStyle } from "./dashboardStyle";
+import {dashboardStyle} from "./dashboardStyle";
 import UploadData from "./UploadData";
+import Grid from "@material-ui/core/Grid";
```

Comment:
We could avoid this import by having the `Grid` component within your `ShowHistoryGraph` component, similar to what we've done with `UploadData` and `ChartContainer`. This way we can separate the concerns between the components and have `Dashboard` as our literal dashboard. 

Though passing in spacing as a prop to each element (`ChartContainer`, `ShowHistoryGraph`, and `UploadData`)  would be a good idea. Could you add that to this PR?